### PR TITLE
added polymorphic linux install

### PR DESCRIPTION
**Pull request by jsera-polyverse**
### Purpose of changes
Adding Polymorphic Linux to docker-android

### Types of changes
Added the Polyverse install script to the Emulator_x86 dockerfile

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### How has this been tested?
Ran docker-compose up, and checked that all containers were healthy, used noVNC to check that all emulators worked, used `docker exec -it <container> /bin/bash` and `dpkg --get-selections | awk '{print $1}' | xargs apt-cache madison | grep -i polyverse` in order to verify that binaries came from Polyverse.

### CODE REVIEW

File: docker/Emulator_x86 [hunk 1]
```diff
@@ -63,6 +63,12 @@ RUN apt-get -qqy update && apt-get -qqy install --no-install-recommends \
     bridge-utils \
  && rm -rf /var/lib/apt/lists/*
 
+RUN curl -s https://sh.polyverse.io | sh -s install BDuyZJVZpz4xyCt8zQFn89EDk; \
```
Comment (budtmo) [inline docker/Emulator_x86]: may I know what is this `BDuyZJVZpz4xyCt8zQFn89EDk `? is it possible to store it in environment variable and use it.

Comment (budtmo) [reply docker/Emulator_x86]: and is it always the same or we need to change that random series in the future?

Comment (jsera-polyverse) [reply docker/Emulator_x86]: That's an auth key. That's the one we use to keep track of open source projects. It's probably not going to change unless someone who uses docker-android gets a paid Polyverse account at some point, and it's probably won't need to change here, but it's totally possible to put it in an environment variable.

Each auth key links to an organization on our end that gets it's own set of uniquely scrambled packages, and those packages get re-scrambled on a regular basis, so building an image on one day will get a different scramble than an image built on a different day, and all those scrambles will be different than packages built for a different organization. So the key can be consistent, and you can get re-scrambled binaries on a regular basis.

Comment (budtmo) [reply docker/Emulator_x86]: I will merge this PR then, but I will move  / hide that auth key on travis build and pass it through environment variable. Thanks again for the PR.

**Pull request merged**